src/Shared/Traits.cs

```diff
@@ -196,6 +196,11 @@ public bool LogTaskInputs
         /// </summary>
         public readonly bool DisableNuGetSdkResolver = Environment.GetEnvironmentVariable("MSBUILDDISABLENUGETSDKRESOLVER") == "1";
 
+        /// <summary>
+        /// Don't delete TargetPath metadata from associated files found by RAR.
+        /// </summary>
+        public readonly bool TargetPathForRelatedFiles = Environment.GetEnvironmentVariable("MSBUILDTARGETPATHFORRELATEDFILES") == "1";
```

Comment:
This feels a little like a double-negative; maybe change this to `...== "";` here and remove the `!` in ReferenceTable.cs?

Reply:
I was trying to avoid a double negative in the name.